sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.NUnit.cs

```diff
@@ -282,6 +282,21 @@ public void TestCaseSource14()
         public void TestCaseSource15() // Don't raise on skipped test methods
         {
         }
+
+        private static readonly TestCaseData[] Cases =
+        {
+          new TestCaseData("foo", "bar").Returns(false),
+          new TestCaseData(100, 100).Returns(true),
+        };
+
+        // See: https://github.com/SonarSource/sonar-dotnet/issues/3751
+        // Returned value is verified by the framework and no assertion is needed.
+        [Test]
+        [TestCaseSource(nameof(Cases))] 
+        public bool Test(object obj1, object obj2) // Noncompliant - FP
+        {
```

Comment:
Can you please add two more testcases with `void` return type? One TP without an assert. One compliant with assert. Just to be sure we don't ignore all methods with that attribute.

Reply:
As far as I can tell both these cases were already covered in the test cases above. See lines 210 and 217.